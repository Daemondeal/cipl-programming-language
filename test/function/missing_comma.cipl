proc f(a, b c, d, e):
    pass