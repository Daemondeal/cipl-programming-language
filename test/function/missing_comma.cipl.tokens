Token          Lexeme         Line 
PROC           proc           1    
IDENTIFIER     f              1    
LEFT_PAREN     (              1    
IDENTIFIER     a              1    
COMMA          ,              1    
IDENTIFIER     b              1    
IDENTIFIER     c              1    
COMMA          ,              1    
IDENTIFIER     d              1    
COMMA          ,              1    
IDENTIFIER     e              1    
RIGHT_PAREN    )              1    
COLON          :              1    
LINE_END       \n             2    
INDENT                        2    
PASS           pass           2    
LINE_END       \n             2    
DEDENT                        2    
EOF                           2    
