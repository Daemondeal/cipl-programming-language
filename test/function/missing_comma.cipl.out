::stdout::

::stderr::
Error on line 1 at `c`: Expect ')' after parameters
